src/app/public/modules/angular-tree/angular-tree-node.component.html

```diff
@@ -0,0 +1,115 @@
+<div
+  role="treeitem"
+  [attr.aria-current]="node.isActive"
+  [attr.aria-expanded]="node.isExpanded"
+  [attr.aria-selected]="ariaSelected()"
+  [class]="node.getClass()"
+  [class.tree-node]="true"
+  [class.tree-node-active]="node.isActive"
+  [class.tree-node-collapsed]="node.isCollapsed && node.hasChildren"
+  [class.tree-node-expanded]="node.isExpanded && node.hasChildren"
+  [class.tree-node-focused]="node.isFocused"
+  [class.tree-node-leaf]="node.isLeaf"
```

Comment:
I'm also curious if having only one binding (`ngClass`) is more performant than having several `class` bindings.

Reply:
This is taken from the original API's template.
https://angular2-tree.readme.io/docs/templates#treenodefulltemplate
Since we were creating our own template, I tried not to stray too far from the original, but I'm totally in favor of better performance. What do you think?

Reply:
Okay that's fine. Can't say I like it though!